Dockerfile-linux.template

```diff
@@ -307,10 +307,23 @@ RUN set -eux; \
 		"pip==$PYTHON_PIP_VERSION" \
 {{ if .setuptools then ( -}}
 		"setuptools==$PYTHON_SETUPTOOLS_VERSION" \
-{{ ) else "" end -}}
+{{ ) else ( -}}
+		--no-setuptools \
+		--no-wheel \
+{{ ) end -}}
 	; \
 	rm -f get-pip.py; \
 	\
-	pip --version
+	pip --version; \
+{{ if .setuptools then ( -}}
+	# smoke test to ensure that setuptools/wheel are included
```

Comment:
Longer term should these smoke tests live in `docker-library/official-images` where the other tests are? eg:
https://github.com/docker-library/official-images/tree/master/test/tests/python-imports
https://github.com/docker-library/official-images/tree/master/test/tests/python-no-pyc

Reply:
Yeah, that makes sense. I removed them so they can just be in the test scripts.